sample/strategy.py

```
@@ -47,29 +60,40 @@ def get_65_by_contract() -> dict[pd.Period, pd.Series]:
+K_EXIT = "Exit"
+K_PRICING_DATE_LIST = "Pricing Date List"
+K_TC5_M2 = "TC5 M2"
+K_M1_T = "M1(T)"
+K_PREDICTED_M1_T1 = "Predicted M1(T+1)"
 
 
-def strategy(exit_option: ExitOption, buffer: float, predict_file: str) -> List[SimpleTrade]:
+def strategy(
+    exit_option: ExitOption, buffer: float, predict_file: str
+) -> List[SimpleTrade]:
     assert exit_option == EXIT_NON_PRICING
```

Ruff (0.12.2)

Use of `assert` detected

(S101)